### ISSUE TITLE
Testnet 3 Docs

### ISSUE THREAD

**Issue by avive** [karma-coin/karmacoin-docs#6]
Update testnet docs for testnet 3:

- https://github.com/karma-coin/karmacoin-docs/blob/main/versioned_docs/version-0.1.0/testnet/basics.md
- https://github.com/karma-coin/karmacoin-docs/blob/main/versioned_docs/version-0.1.0/testnet/validator.md
- https://github.com/karma-coin/karmacoin-docs/blob/main/versioned_docs/version-0.1.0/testnet/tn2_parameters.md

**Comment by HolyGrease** [comment]
Do we need to describe how to do basic transactions? For instance, `new user` transactions is very to initiate from UI because user has to get evidence from the verifier. This seems to me unusable, so I think we should remove this transaction as all other because without `new_user` other transaction do not make sense

**Comment by avive** [comment]
no need to describe - we want everybody to sign up via the app

**Comment by avive** [comment]
good idea to remove. For nominators, pool owners, and validators, we should recommend setting social link via the app.
